index.html

```diff
@@ -213,6 +218,7 @@ <h3>Let's connect</h3>
         <ul class="ul-1">
             <li class="li-1">
                 <h3 class="li-1_title">languages <i class='fa fa-chevron-down'></i></h3>
+                <p></p>
```

Comment:
- [ ] You must replicate the design as much as possible (e.g., font, colors, images, tests, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
- [ ] Your drop-down menu should be at the right of the page as the Figma design. Kindly implement it.
### Your design
![his](https://user-images.githubusercontent.com/81727988/171597690-3a3c1e2e-cefc-4d05-9a42-8aa987a3ca93.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/171597749-4d9f5acf-e9d1-496c-ba41-0e4f54927f3f.png)